README.md

```diff
@@ -617,130 +621,143 @@ Note that it is also possible to generate a synthetic dataset based on the entir
 
 There are many options for training the model with your synthetic data-enhanced dataset.
 
-> **Note:** **Every** `ilab` command needs to run from within your Python virtual environment.
+### ✋ Before you begin training
 
-#### ilab model training pipelines
+There are a few models you need to download before running the InstructLab end-to-end workflow locally.
 
-`ilab model train` has three pipelines: `simple`, `full`, and `accelerated`. The default is `accelerated`.
+- Download the `granite-7b-lab` model for training:
```

Comment:
Technically, users could train other models - I'd just change the wording to say something like "for example"

Reply:
True, I only added this one because granite-7b-lab is the default model in the config. So the user should download it at some point. 

This part is more suppose to be like a Prereq

Reply:
ehhhhhhhhh I dont think anything other than simple really works with non-granite models

Reply:
Yeah let's just leave this then, for now